drivers/soundwire/bus.c

```diff
@@ -1276,23 +1276,12 @@ int sdw_configure_dpn_intr(struct sdw_slave *slave,
 	return ret;
 }
 
-static int sdw_slave_set_frequency(struct sdw_slave *slave)
+u8 sdw_slave_get_scale_index(struct sdw_slave *slave, u8 *base)
```

Comment:
commit message:

when the 4th stream is open -> opened?
update the sacle index if needed. -> sample?